ddtrace/appsec/_common_module_patches.py

```diff
@@ -0,0 +1,134 @@
+import ctypes
+import gc
+from typing import Any
+from typing import Callable
+from typing import Dict
+
+from ddtrace.internal.logger import get_logger
+from ddtrace.settings.asm import config as asm_config
+from ddtrace.vendor.wrapt import FunctionWrapper
+from ddtrace.vendor.wrapt import resolve_path
+
+
+log = get_logger(__name__)
+_DD_ORIGINAL_ATTRIBUTES: Dict[Any, Any] = {}
+
+
+def patch_common_modules():
+    try_wrap_function_wrapper("builtins", "open", wrapped_open_CFDDB7ABBA9081B6)
+
+
+def unpatch_common_modules():
+    try_unwrap("builtins", "open")
+
+
+def wrapped_open_CFDDB7ABBA9081B6(original_open_callable, instance, args, kwargs):
```

Comment:
Curious about the name of this function, does it have any meaning?

Reply:
no meaning, it's just useful to have a specific unique name to easily truncate the trace send to the customer up to that function when an exploit will be detected. (We don't want to send unnecessary information about our instrumentation)

Reply:
We iterate through frames for IAST here:
https://github.com/DataDog/dd-trace-py/blob/3d1e9c9960b351fbf1af0e218d0e88ffdde9f5ec/ddtrace/appsec/_iast/_stacktrace.c#L94-L95

Reply:
I prefer to use a "tag name", it's more precise than a file for that particular truncation.